server/util/getMetrics.js

```diff
@@ -74,35 +78,44 @@ const countUnexpectedBehaviors = ({
     return countScenarioResult(scenarioResult);
 };
 
```

Comment:
Not unique to this PR but curious why these utilities get copied across client and server. If they were pure functions, would they be usable across both since we have the monorepo?